package.json

```diff
@@ -45,7 +45,10 @@
     "build-storybook": "yarn run install-storybook && build-storybook",
     "install-storybook": "./scripts/storybook-install",
     "remove-storybook": "./scripts/storybook-install -d",
-    "remove-storybook-deps": "./scripts/storybook-install -r"
+    "remove-storybook-deps": "./scripts/storybook-install -r",
+    "install-docusaurus": "cd docusaurus/ && yarn install",
+    "build-docusaurus": "cd docusaurus/ && yarn build",
+    "docusaurus": "cd docusaurus/ && yarn start"
```

Comment:
Could you add these informations somewhere in the documentation, since we are quite in topic? 😅

Reply:
Information in the README.